### DISCUSSION TITLE
[Help]: Unable to get `jump: true` option to work

### DISCUSSION THREAD

**Discussion by henrybabbage** [davidjerleke/embla-carousel#793] [Help] [answered]
### Summary

Hello,

First of all, thank you for the great library!

My issue I am facing is that I am trying to set the `jump` option to true in order to remove the motion from the slide transition. So far I have added jump: true in the scrollNext and scrollPrev methods without luck, and I also tried it in the Autoplay plugin options without luck.

A codesandbox reproduction of the issue is linked below.

Many thanks in advance!

### If applicable, which variants of Embla Carousel are relevant to this question?

- [X] embla-carousel **(Core)**
- [X] embla-carousel-react
- [ ] embla-carousel-vue
- [ ] embla-carousel-svelte
- [X] embla-carousel-autoplay
- [ ] embla-carousel-auto-scroll
- [ ] embla-carousel-solid
- [ ] embla-carousel-auto-height
- [ ] embla-carousel-class-names
- [ ] embla-carousel-docs **(Documentation)**
- [ ] embla-carousel-docs **(Generator)**

### CodeSandbox example

[CodeSandbox](https://codesandbox.io/p/sandbox/embla-carousel-autoplay-react-forked-g7tn2g?layout=%257B%2522sidebarPanel%2522%253A%2522EXPLORER%2522%252C%2522rootPanelGroup%2522%253A%257B%2522direction%2522%253A%2522horizontal%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522id%2522%253A%2522ROOT_LAYOUT%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522cltrcemr400063b6egodz2n3g%2522%252C%2522sizes%2522%253A%255B100%252C0%255D%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522EDITOR%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522id%2522%253A%2522cltrcemr400023b6eecqbqgje%2522%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522SHELLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522id%2522%253A%2522cltrcemr400033b6ey5t4vazm%2522%257D%255D%252C%2522sizes%2522%253A%255B100%255D%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522DEVTOOLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522id%2522%253A%2522cltrcemr400053b6evoaqkild%2522%257D%255D%252C%2522sizes%2522%253A%255B100%255D%257D%255D%252C%2522sizes%2522%253A%255B50%252C50%255D%257D%252C%2522tabbedPanels%2522%253A%257B%2522cltrcemr400023b6eecqbqgje%2522%253A%257B%2522id%2522%253A%2522cltrcemr400023b6eecqbqgje%2522%252C%2522tabs%2522%253A%255B%255D%257D%252C%2522cltrcemr400053b6evoaqkild%2522%253A%257B%2522id%2522%253A%2522cltrcemr400053b6evoaqkild%2522%252C%2522activeTabId%2522%253A%2522cltrcmtwl00e83b6eexsky646%2522%252C%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522cltrcemr400043b6e6wbykjy4%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522UNASSIGNED_PORT%2522%252C%2522port%2522%253A0%252C%2522path%2522%253A%2522%252F%2522%257D%252C%257B%2522type%2522%253A%2522SANDBOX_INFO%2522%252C%2522id%2522%253A%2522cltrcmtwl00e83b6eexsky646%2522%252C%2522mode%2522%253A%2522permanent%2522%257D%255D%257D%252C%2522cltrcemr400033b6ey5t4vazm%2522%253A%257B%2522tabs%2522%253A%255B%255D%252C%2522id%2522%253A%2522cltrcemr400033b6ey5t4vazm%2522%257D%257D%252C%2522showDevtools%2522%253Atrue%252C%2522showShells%2522%253Afalse%252C%2522showSidebar%2522%253Atrue%252C%2522sidebarPanelSize%2522%253A15%257D)

**Comment by sarussss** [comment]
Hi @henrybabbage 
In my opinion, [jump](https://www.embla-carousel.com/plugins/autoplay/#jump) still works normally.
"When set to true, autoplay will do instant slide transitions when advancing."
Use
scrollNext(true)
scrollPrev(true)
to go to the next slide instantly.

**Reply by henrybabbage** [reply]
Oh, is the syntax scrollNext(true) ? Not, scrollNext({ jump: true }) ?

**Reply by sarussss** [reply]
yeah

**Reply by sarussss** [reply, marked as answer]
emblaApi.scrollPrev(true)

**Reply by davidjerleke** [reply]
@sarussss thanks a ton for helping out 👍🏻. 

@henrybabbage please don’t forget to click **Mark as answer** on the comment that holds the answer to your question. This will help other devs.

Best,
David

**Reply by sarussss** [reply]
Hi @davidjerleke.
Actually you don't need to thank me, I'm the one who has to do it, because you're spending your free time to make something great for everyone, I just want to contribute a little within my ability. to save you some time if possible.
Thanks millions!

**Reply by henrybabbage** [reply]
Thank you both so much for help yesterday! And with such fast responses :)
